StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetVerticalViewController.swift

```diff
@@ -40,17 +41,24 @@ class PaymentSheetVerticalViewController: UIViewController, FlowControllerViewCo
                 return nil
             }
             return paymentMethodFormViewController.paymentOption
-
         }
     }
-    var selectedPaymentMethodType: PaymentSheet.PaymentMethodType?
+    var selectedPaymentMethodType: PaymentSheet.PaymentMethodType? {
```

Comment:
I think we need to cover the weird edge case w/ Link in the wallet?

Reply:
If so, can be done before/after https://github.com/stripe/stripe-ios/pull/3643 lands.